api/base/serializers.py

```diff
@@ -120,13 +118,15 @@ def to_representation(self, value):
         the link is represented as a links object with 'href' and 'meta' members.
         """
         url = super(JSONAPIHyperlinkedIdentityField, self).to_representation(value)
+        if not url:
+            raise SkipField
 
         meta = {}
         for key in self.meta or {}:
             if key == 'count':
```

Comment:
It seems like this block only acts on a particular key. What is the advantage of iterating over the entire dictionary instead of just accessing `self.meta['count']` directly?